src/pmi/pmi.cs

```diff
@@ -199,9 +200,10 @@ public override void FinishAssembly(Assembly assembly)
             $"Completed assembly {assemblyName} - #types: {typeCount}, #methods: {methodsPrepared}, " +
             $"skipped types: {uninstantiableTypeCount}, skipped methods: {uninstantiableMethodCount}");
 
-        if (_verbose)
+        if (_time || _verbose)
         {
-            Console.WriteLine($", elapsed ms: {elapsed.TotalMilliseconds:F2}");
+            string prefix = _time && !_verbose ? "" : ", ";
+            Console.WriteLine($"{prefix}Time: {elapsed.TotalMilliseconds:F2}ms");
```

Comment:
So in the `_time && !_verbose` case there is no ", "? doesn't that just jam it against the skipped methods count?

Reply:
Yeah, not sure what I was thinking here. Will fix.